nixos/tests/systemd-portabled.nix

```diff
@@ -0,0 +1,51 @@
+import ./make-test-python.nix ({pkgs, lib, ...}: let
```

Comment:
Can you add this to the nixosTests attribute set, too?

Reply:
Ah, hadn't seen that one before tbh, was looking for a `default.nix` but didn't see `all-tests.nix` up until now.